### ISSUE TITLE
The "kitchensink" button cannot be removed from the toolbar in the classic block

### ISSUE THREAD

**Issue by azaozz** [WordPress/gutenberg#10963]
Happens because 'kitchensink' is hard-coded after the `'mce_buttons'` filter. The corresponding button, `wp-adv` is easily removable.

To test, add something like that to a plugin:
```php
add_filter( 'mce_buttons', 'my_mce_buttons' );
function my_mce_buttons( $buttons ) {
	$buttons = array_diff( $buttons, array( 'kitchensink' ) );
	return $buttons;
}
```

In addition to not hard-coding that button, a little bit of js needs to be added to always show all editor toolbars in the classic block when the button is not present. Also, it would have been nice if the classic block would "remember" the state of the last use of that button, perhaps in v2 (would need REST API support, etc.).

**Comment by draganescu** [comment]
There is also a comment in the code:
`// TODO: the following is for back-compat with WP 4.9, not needed in WP 5.0. Remove it after the release.`

**Comment by Mamaduka** [comment]
Fixed via #10964.
